### PR TITLE
fix(update): add config update liferay from 7.3 to 7.4

### PR DESCRIPTION
**Pull request by tuannn2**
- Update document for upgrade liferay from 7.3 to 7.4
Suggest Reviewer: @KoukiHama

### CODE REVIEW

**Comment by KoukiHama** [comment]
Plan to review this on coming Monday.

File: content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md [hunk 1]
```diff
@@ -53,60 +53,63 @@ Link contains source: <https://github.com/eclipse/sw360.git>
...
+
+## Liferay Database Migration
+
+
+* Go to `liferay-ce-portal-7.4.3.18-ga18/tools/portal-tools-db-upgrade-client` folder
+
+* Edit `app-server.properties` to add the following parameters:
+
+```
+  dir={LIFRAY_PATH_7.4}/tomcat-9.0.56
```
Comment (heliocastro) [inline content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md:72]: Typo: LIFRAY

File: content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md [hunk 2]
```diff
@@ -53,60 +53,63 @@ Link contains source: <https://github.com/eclipse/sw360.git>
...
-    company.security.strangers=false
-    company.security.strangers.with.mx=false
-    company.security.strangers.verify=false
+* Edit `portal-upgrade-database.properties` to add the following parameters:
+
+```
+  jdbc.default.driverClassName=org.postgresql.Driver
+  jdbc.default.url=jdbc:postgresql://{POSTGRE_HOST}:5432/lportal
+  jdbc.default.username={POSTGRE_USER}
+  jdbc.default.password={POSTGRE_PASSWORD}
```
Comment (heliocastro) [inline content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md:85]: Typo ? POSTGRE
Should not be POSTGRES ?

File: content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md [hunk 3]
```diff
@@ -156,30 +178,40 @@ Link contains source: <https://github.com/eclipse/sw360.git>
...
   `curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py`
 
-   if there is no proxy, skip option `--proxy=http://username:password@hostname`
+   if there is no proxy, skip option `--proxy=http://username:password@hostname:port`
 
      ```bash
-        $ sudo python2 get-pip.py --proxy=http://username:password@hostname
+        $ sudo python2 get-pip.py --proxy=http://username:password@hostname:port
         $ pip --version
      ```
```
Comment (heliocastro) [inline content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md:186]: Move to python 3

**Review by heliocastro — CHANGES_REQUESTED** [review]

File: content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md (hunk 3 above)
Comment (tuannn2) [reply content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md:186]: The migration scripts  are written in `python2` and depend on `python2-couchdb. 
 https://github.com/eclipse-sw360/sw360/blob/ec28fa1b4f032ef723b3fa7349cbed3d8bf79122/scripts/migrations/README.md?plain=1#LL5C132-L5C132

Comment (ag4ums) [reply content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md:186]: @tuannn2 the page is not updated, as we have those old python2 migration scripts, but as python2 is no more supported, all our migration scripts form rel 16 onwards are in python3

Comment (tuannn2) [reply content/en/docs/Deployment/Upgrading/Upgrade_Version_SW360_16_To_17.md]: Thanks for your comments, @heliocastro  @ag4ums. I updated.

**Review by KoukiHama — CHANGES_REQUESTED** [review]
> sql.data.com.liferay.portal.kernel.model.Country.country.id=21529
 
 Whether it in sw360.property is unnecessary or necessary, as it exists in the official main branch, it might be good to have some reference to it on upgrade documents.
 
 about re-import lars, please note whether user need to delete old file or not.

**Review by KoukiHama — APPROVED** [review]

**Pull request merged**